keyboards/ploopyco/trackball_nano/keymaps/default/keymap.c

```diff
@@ -20,3 +20,11 @@
 
 // Dummy
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {{{ KC_NO }}};
+
+void suspend_power_down_user(void) {
+    adns5050_power_down();
```

Comment:
Perhaps a comment like this would be helpful.
```suggestion
    adns5050_power_down(); // disables waking host by trackball
```

Reply:
Good idea, thanks.